test/client/views/requestDetails.js

```diff
@@ -10,9 +13,24 @@ var internals = {};
 
 describe('RequestDetailsView', function () {
 
+    beforeEach( function () {
+
+        this.model = new Backbone.Model();
+        this.model.hasError = function() {};
```

Comment:
So, this is something I've probably missed here and other places. I think I would prefer to use the actual model in this case and use sinon's stub functionality. The benefit of using stub vs just attaching a function is that stub will through an exception if the method it's trying to stub doesn't exist. This helps check that there wasn't a typo or that the contract between the model and the view didn't change.

Something like this (although I didn't actually test that this code works):
```
this.model = new Request();
sinon.sub(this.model, 'hasError');
```